### ISSUE TITLE
fix DCGAN

### ISSUE THREAD

**Issue by twairball** [twairball/tensor2gan#8]
- [x] clip gradients [Ref](https://stackoverflow.com/questions/36498127/how-to-effectively-apply-gradient-clipping-in-tensor-flow)
- [x] `0.5 * learning_rate` for Discriminator
- [x] add `alpha=0.9` to `d_loss_fake` labels
